cmd/sdk-server/main.go

```diff
@@ -25,8 +25,10 @@ import (
 	"strings"
 	"time"
 
+	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	gwruntime "github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
@markmandel it looks like this is the same package that we use in our other controllers. Is there a reason we use this one over https://pkg.go.dev/log/slog?

Reply:
Slog didn't exist when we started this project 😄